src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ImportDocumentSnapshot.cs

```diff
@@ -48,6 +48,9 @@ public async Task<SourceText> GetTextAsync()
         return _sourceText;
     }
 
+    public Task<RazorCodeDocument> GetGeneratedOutputAsync(bool _)
```

Comment:
Did you purposefully leave out the parameter name here?

Reply:
Yes, since it would be unused anyway. Would you prefer to have it? I don't feel strongly either way.